internal/manifest/exohcl/components.go

```diff
@@ -154,29 +153,6 @@ func (c *Component) Analyze(ctx *AnalysisContext) {
 	} else {
 		c.Spec, _ = AnalyzeString(ctx, specAttr.Expr)
 	}
-
-	depsAttr := content.Attributes["depends_on"]
```

Comment:
Don't we still need this? How otherwise does a user explicitly define a dependency? How would this docker compose file work?

```docker-compose.yaml
services:
  t0:
    image: bash
    command: "sleep infinity"
  t1:
    image: bash
    command: "sleep infinity"
    depends_on:
      - "t0"
```

Reply:
depends_on works via https://github.com/deref/exo/pull/436 now - This is simply removing the ability to provide extra/explicit dependencies in the manifest, which I decided isn't really an important feature yet (and if it becomes one, might have to work differently, merging dependencies with the spec analysis, or similar).